js/spa/test/image_editor_events.spec.ts

```diff
@@ -9,61 +9,44 @@ test("upload events work as expected", async ({ page }) => {
 });
 
 test("change events work as expected", async ({ page }) => {
-	await page.getByLabel("Upload button").first().click();
-	const uploader = page.locator("input[type=file]").first();
-	await uploader.setInputFiles(["./test/files/cheetah1.jpg"]);
-
 	const change_text = page.locator("#change h2");
-	await expect(change_text).toContainText("1");
```

Comment:
split up the tests here because playwright was being weird. Coverage is the same now though.